product-service/src/functions/createProduct/handler.ts

```diff
@@ -0,0 +1,34 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import createClient from 'src/utils/createClient';
+
+import schema from './schema';
+
+const createProduct: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+  console.log(JSON.stringify(event));
+  const client = createClient();
+  try {
+    client.connect();
+    await client.query('BEGIN')
+    const { id, title, description, price, count } = event.body;
```

Comment:
"Id" should not be passed as input parameter to create product. "Id" field should be created db and will be output parameter for db and as a result in response